### PR TITLE
Add base providersService

### PR DESCRIPTION
**Pull request by alexmckenley**
@deini @malucky What do you think about this for a base providersService?

### CODE REVIEW

**Comment by deini** [comment]
pinging @ssalazars

File: public/_common/services/providers/providers.service.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+  angular
+    .module('ammo.services.providers', [
+      'ammo.services.providers.youtube'
+    ])
+    .factory('providersService', providersService);
+
+    function providersService($log, youtubeService) {
+      var providers = {
+            youtube: youtubeService
```
Comment (deini) [inline public/_common/services/providers/providers.service.js]: Looks inconsistent how you declare and set providers but not service. I don't know exactly how the styleguide manages this but I think that if we are going to do a "complex" declaration we should just declare it and set it in another line. /discuss

Comment (alexmckenley) [inline public/_common/services/providers/providers.service.js]: I see what you are saying... Should I move this below the declaration, or move the service definition up here as well?

Comment (deini) [inline public/_common/services/providers/providers.service.js]: If this is not defined in the styleguide then just go with whatever looks better :lipstick:

**Comment by alexmckenley** [comment]
@deini, can you have a look at the new style?

**Comment by deini** [comment]
@alexmckenley Style looks good now. Waiting for new folder structure + file renaming after our discussion @ teamspeak

**Comment by deini** [comment]
:+1: LGTM

**Pull request merged**